Msys2 CI failing on gettext

Steps to reproduce
------------------
Run any `master` (or unbroken branch) build


Expected Output
---------------
No errors for msys2 job


Actual Output
-------------
Errors around gettext
```
gdist\gettextutil.py:277: in check_po
    subprocess.check_output(
D:\a\_temp\msys64\mingw64\lib\python3.9\subprocess.py:424: in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
D:\a\_temp\msys64\mingw64\lib\python3.9\subprocess.py:528: in run
    raise CalledProcessError(retcode, process.args,
E   subprocess.CalledProcessError: Command '['msgfmt', '--check-format', '--check-domain', 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmpqom77s0r.po', '-o', 'nul']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:
tests\test_po.py:46: in test_potfile_format
    gettextutil.check_pot(pot_path)
gdist\gettextutil.py:299: in check_pot
    check_po(po_path, ignore_header=True)
gdist\gettextutil.py:281: in check_po
    raise GettextError(e.output)
E   gdist.gettextutil.GettextError: msgfmt: error while opening "nul" for writing: Invalid argument
```
e.g. https://github.com/quodlibet/quodlibet/runs/4754024878?check_suite_focus=true

--- a/gdist/gettextutil.py
+++ b/gdist/gettextutil.py
@@ -1,5 +1,5 @@
 # Copyright 2015-2017 Christoph Reiter
-#             2019-21 Nick Boultbee
+#             2019-22 Nick Boultbee
 #
 # Permission is hereby granted, free of charge, to any person obtaining
 # a copy of this software and associated documentation files (the
@@ -215,7 +215,7 @@ def po_stats(po_path: Path):
 
     try:
         return subprocess.check_output(
-            ["msgfmt", "--statistics", str(po_path), "-o", os.devnull],
+            ["msgfmt", "--statistics", str(po_path), "-o", "-"],
             universal_newlines=True, stderr=subprocess.STDOUT).strip()
     except subprocess.CalledProcessError as e:
         raise GettextError(e.output)
@@ -275,7 +275,7 @@ def check_po(po_path: Path, ignore_header=False):
     check_arg = "--check" if not ignore_header else "--check-format"
     try:
         subprocess.check_output(
-            ["msgfmt", check_arg, "--check-domain", str(po_path), "-o", os.devnull],
+            ["msgfmt", check_arg, "--check-domain", str(po_path), "-o", "-"],
             stderr=subprocess.STDOUT, universal_newlines=True)
     except subprocess.CalledProcessError as e:
         raise GettextError(e.output)

--- a/gdist/gettextutil.py
+++ b/gdist/gettextutil.py
@@ -214,9 +214,8 @@ def po_stats(po_path: Path):
     """Returns a string containing translation statistics"""
 
     try:
-        return subprocess.check_output(
-            ["msgfmt", "--statistics", str(po_path), "-o", "-"],
-            universal_newlines=True, stderr=subprocess.STDOUT).strip()
+        return subprocess.run(["msgfmt", "--statistics", str(po_path)],
+                              universal_newlines=True).stderr.strip()
     except subprocess.CalledProcessError as e:
         raise GettextError(e.output)
 
@@ -274,9 +273,8 @@ def check_po(po_path: Path, ignore_header=False):
 
     check_arg = "--check" if not ignore_header else "--check-format"
     try:
-        subprocess.check_output(
-            ["msgfmt", check_arg, "--check-domain", str(po_path), "-o", "-"],
-            stderr=subprocess.STDOUT, universal_newlines=True)
+        subprocess.run(["msgfmt", check_arg, "--check-domain", str(po_path)],
+                       universal_newlines=True).stderr
     except subprocess.CalledProcessError as e:
         raise GettextError(e.output)
 

